content/en/docs/Installation/_index.md

```diff
@@ -18,6 +18,7 @@ Currently, Kyverno runs as a single replica in the `Deployment` resource. Suppor
 |--------------------------------|----------------|----------------|
 | 1.2.1                          | 1.14           | 1.17           |
 | 1.3.0                          | 1.16           | 1.20*          |
+| 1.4.0                          | 1.16           | 1.20           |
```

Comment:
1.4.0 won't support 1.21? Seems like it should at this point.

Reply:
We can add it if we are sure that it supports 1.21. Did you test this release with 1.21?

Reply:
e2e tests are currently using KinD 0.10 which maps to Kubernetes 1.20. [This issue](https://github.com/kyverno/kyverno/issues/1973) created to bump to 1.21.

Reply:
I think we are good to go with adding 1.21 @chipzoller